docs/src/main/asciidoc/_admin-guide/Management_resources.adoc

```diff
@@ -285,6 +285,55 @@ See <<descriptions,Descriptions>>
 below for how to learn more about the attributes a particular resource
 exposes.
 
+[[overridden-attribute-value]]
+=== Override Attribute Value with an Environment Variable
+
+It is possible to override the value of any attribute by providing an Environment Variable with a name
```

Comment:
s/any attribute/any simple attribute/g

And then somewhere a discussion of what a simple attribute is (not a ModelType LIST, OBJECT, PROPERTY.)

Reply:
done, thanks